### PR TITLE
Clarification question changes for DOS4

### PR DESCRIPTION
**Pull request by katstevens**
Trello: https://trello.com/c/3U1ORXqu/568-dos4-clarification-question-content-improvements

We're removing the post-clarification-deadline 'followup' question functionality. Instead we are directing suppliers to the Help page where they can contact support. This new process has been agreed with CCS.

clarification-content

Stefan has also supplied some new content that should help reduce the number of questions we receive (see ticket).

### CODE REVIEW

File: app/templates/frameworks/updates.html [hunk 1]
```diff
@@ -107,24 +105,36 @@
...
+              <a href="{{ url_for('external.help') }}">Get help if there's a problem with your account</a>
+            </p>
             <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
               {%
                 with
                 large=true,
-                question = "Ask a clarification question",
                 name = clarification_question_name,
-                hint = "You should ask any questions you have about {} here. The Crown Commercial Service will not respond to you individually. (Maximum 5000 characters per question.)".format(framework.name),
+                hint = "(Maximum 5000 characters per question)",
```
Comment (lfdebrux) [inline app/templates/frameworks/updates.html]: Minor quibble, but we could remove the parentheses from here?

File: tests/app/main/test_frameworks.py [hunk 2]
```diff
@@ -3493,7 +3493,7 @@ def test_question_box_is_shown_if_countersigned_agreement_is_not_yet_returned(se
         data = response.get_data(as_text=True)
 
         assert response.status_code == 200
-        assert u'Ask a question about your G-Cloud 7 application' in data
+        assert u'Contact the support team' in data
```
Comment (lfdebrux) [inline tests/app/main/test_frameworks.py]: We can remove the `u`

File: app/templates/frameworks/updates.html (hunk 1 above)
Comment (katstevens) [reply app/templates/frameworks/updates.html]: I'll check with Stefan.

**Comment by katstevens** [comment]
Functional tests here: https://github.com/alphagov/digitalmarketplace-functional-tests/pull/661

File: app/templates/frameworks/updates.html (hunk 1 above)
Comment (katstevens) [reply app/templates/frameworks/updates.html]: Stefan says remove 'em. Commit incoming

**Pull request merged**